src/Client/ClientBase.cpp

```diff
@@ -850,18 +851,8 @@ void ClientBase::setDefaultFormatsAndCompressionFromConfiguration()
             default_input_compression_method = chooseCompressionMethod(*file_name, "");
     }
 
-    format_max_block_size = getClientConfiguration().getUInt64("format_max_block_size", global_context->getSettingsRef()[Setting::max_block_size]);
```

Comment:
`format_max_block_size` is not used - OK (though consider would vote against making cleanup changes for PRs that will be backported)

Reply:
(Hopefully won't be backported.)